lid: add page (#7578)
tldr-pages/tldr: pages/linux/lid.md

# lid

> > NOTE: This page is currently a redirection stub. If you are familiar with this program, please open a pull request!

Reviewer: ```suggestion
> NOTE: This page is currently a redirection stub. If you are familiar with this program, please open a pull request.
```
This should fix `tldr-bot`'s error.

Author: I think there is a case to be made for an exception here. If it's up to me, I would like to dismiss this CI error and merge anyway.

Author: Okay sure. We're pretty much bikeshedding at this point.

Author: Oh okay, I did not know that. I thought it was just an advisory sort of thing.

# lid

> Display a user's groups or a group's users.

> > More information: <https://manned.org/lid.8>.

Reviewer 2: ```suggestion
> More information: <https://manned.org/lid>.
```

Author: There exists `lid.3` which is completely unrelated. I think we should keep the section number.

> - Show a user's groups (primary and secondary):
> 
> `sudo lid {{user}}`

Reviewer 3: ```suggestion
- List primary and secondary groups of a specific user:

`sudo lid {{username}}`
```

There are several similar items (groups) printed, so `list` is better sounds here.

> - Show a group's users:
> 
> `sudo lid --group {{group}}`

Reviewer 3: ```suggestion
- List users of a specific group:

`sudo lid --group {{group_name}}`
```

# lid

> Query ID database and report results.
> On Fedora and Arch Linux, the binary name `lid` is taken by another program. See `tldr libuser-lid`.

> > More information: <https://example.org/>.

Author: Also, I couldn't find `id-utils`'s `lid` on [manned.org](https://manned.org/browse/search?q=lid). [die.net](https://linux.die.net/man/1/lid)'s page is out of date, and I really do not want to link [unix.com](https://www.unix.com/man-page/debian/1/LID/).

Reviewer 4: ```suggestion
> More information: <https://www.gnu.org/software/idutils/>.
```
Found the GNU version here: https://www.gnu.org/software/idutils/

This link, given it's the homepage thereof, would be preferable over a man page.

It also has a link to a full manual.

Author: Ah thanks for the reminder. I must have missed the notification on @sbrl's reply. Updated.

> - This page is currently a redirection stub:

Reviewer 4: Maybe, a bit of clarification here to make it clear we're looking for contributors if we're merging this early?

```suggestion
- This page is currently a redirection stub (if you are familiar with this version of this command, please open a pull request!):
```

> - This page is currently a redirection stub (if you are familiar with this version of this command, please open a pull request!):

Reviewer 5: ```suggestion
- View documentation for `libuser-lid` (this page is currently a redirection stub; if you are familiar with this version of this command, please open a pull request!):
```
The tldr-bot is throwing an error because "this" ends with the letter "s"

Author: I opted not to use the wording "this version of this command", because it can easily be misunderstood. These are two completely different programs that just so happen to have a naming conflict.
